src/osha/oira/statistics/utils.py

```diff
@@ -141,13 +155,28 @@ def update_assessment(self, country=None):
             .filter(Risk.parent_id.in_(good_module_ids))
             .subquery()
         )
+        return active_risks
+
+    def update_assessment(self, country=None):
+        log.info("Table: assessment")
+        latest_modified_date = (
+            self.session_statistics.query(
+                sqlalchemy.func.max(SurveySessionStatistics.modified)
+            ).first()[0]
+            or datetime.min
+        )
+        if latest_modified_date > datetime.min:
+            log.info(f"Skipping assessments up to and including {latest_modified_date}")
```

Comment:
The same comments I did before about the use of first and the f-string also apply here

Reply:
I love and I hate copy & paste.